genvalidity/src/Data/GenValidity.hs

```diff
@@ -115,13 +115,13 @@ class GenUnchecked a where
     genUnchecked :: Gen a
     default genUnchecked :: (Generic a, GGenUnchecked (Rep a)) =>
         Gen a
-    genUnchecked = to <$> gGenUnchecked
+    genUnchecked = genericGenUnchecked
 
```

Comment:
Renames shouldn't be in the same PR as an added feature.

Reply:
Good point.